views/topic/edit-topic.hbs

```diff
@@ -44,10 +44,9 @@
 
                 <div class="form-group">
                     <label>{{$t "topic._topic.label.content" }}</label>
-
                     <!-- React Magic -->
-                    <div id="content-blocks" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
-                        data-iscoursegroup="{{courseGroupId}}" data-nexboardenabled="{{#ifEnv "FEATURE_NEXBOARD_ENABLED" "true"}}true{{/ifEnv}}"></div>
+                    <div id="content-blocks" data-testid="nexboardButton" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
+                        data-iscoursegroup="{{courseGroupId}}" data-neXboardButtonEnabled="{{#ifEnv "FEATURE_NEXBOARD_BUTTON_ENABLED" "false"}}true{{/ifEnv}}"></div>
```

Comment:
You have a different env var name here and in the config file with the env vars - please remember to add the "D" at the end of the "FEATURE_NEXBOARD_BUTTON_ENABLE" in the app's env vars config.